content/sql/concepts/math-functions/math-functions.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Math Functions'
+Description: 'SQL Math Functions are built-in features in SQL, which let you do mathematical operations, directly in your database queries.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'SQLite'
+  - 'MySQL'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+**SQL Math Functions** are built-in features in SQL, which let you do mathematical operations, such as addition, subtraction, and even more complex calculations, directly in your database queries. This makes it easier and more efficient to work with numerical data in SQL, as you don't need to extract the data and process it separately.
```

Comment:
```suggestion
**SQL Math Functions** are built-in features in SQL, which allow mathematical operations and complex calculations, directly in database queries. This makes it easier and more efficient to work with numerical data in SQL, as there's no need to extract the data and process it separately.
```
No first- or second-person pronouns in Docs.

Also, multiplication, addition, subtraction and division are operators, not functions.